app/decorators/slack_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class SlackDecorator < Draper::Decorator
+  delegate_all
```

Comment:
Are you using a decorator on purpose? It doesn't have much use if you're delegating all methods :-)